packages/ckeditor5-ui/src/index.ts

```diff
@@ -92,6 +92,7 @@ export { default as IframeView } from './iframe/iframeview.js';
 export { default as LabelView } from './label/labelview.js';
 
 export { type LabeledFieldViewCreator, default as LabeledFieldView } from './labeledfield/labeledfieldview.js';
+export { default as SearchTextQueryView } from './search/text/searchtextqueryview.js';
```

Comment:
```suggestion
```

Can't find usage of it in the `ckeditor5-emoji` package. Shouldn't the change be reverted?

Reply:
Done in b53e60f.